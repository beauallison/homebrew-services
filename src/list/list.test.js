jest.mock('child_process');

const list = require('./list');

describe('list()', () =>
  it('should return a Map of services and their status', async () => {
    const output = await list();
    expect(output).toMatchSnapshot();
  }));
