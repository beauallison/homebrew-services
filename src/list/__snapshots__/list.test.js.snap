// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list() should return a Map of services and their status 1`] = `
Map {
  "cassandra" => "stopped",
  "chronograf" => "stopped",
  "elasticsearch" => "stopped",
  "influxdb" => "stopped",
  "kafka" => "stopped",
  "kapacitor" => "stopped",
  "mongodb" => "stopped",
  "rabbitmq" => "stopped",
  "redis" => "started",
  "zookeeper" => "stopped",
}
`;
